server/chat-plugins/datasearch.js

```diff
@@ -614,16 +630,16 @@ function runDexsearch(target, cmd, canAll, message) {
 					break;
 				}
 			}
-
 			if (alts.tiers && Object.keys(alts.tiers).length) {
 				let tier = dex[mon].tier;
 				if (tier[0] === '(' && tier !== '(PU)') tier = tier.slice(1, -1);
+				if (tier === 'New') tier = 'OU';
 				if (alts.tiers[tier]) continue;
 				if (Object.values(alts.tiers).includes(false) && alts.tiers[tier] !== false) continue;
 				// LC handling, checks for LC Pokemon in higher tiers that need to be handled separately,
 				// as well as event-only Pokemon that are not eligible for LC despite being the first stage
 				let format = Dex.getFormat('gen' + maxGen + 'lc');
-				if (!format.exists) format = Dex.getFormat('gen7lc');
+				if (!format.exists) format = Dex.getFormat('gen8lc');
```

Comment:
There is no format called `gen8lc` currently.

Reply:
Since as far as I can tell, this code is supposed to handle cases that won't exist before gen8lc exists, this change should be harmless

Reply:
lc exists now